gatsby/src/pages/borderline-substances/{BnfBorderlineSubstancesTaxonomy.slug}.tsx

```diff
@@ -0,0 +1,132 @@
+import { graphql, Link } from "gatsby";
+import React, { type FC } from "react";
+
+import { Breadcrumbs, Breadcrumb } from "@nice-digital/nds-breadcrumbs";
+import { Grid, GridItem } from "@nice-digital/nds-grid";
+import { PageHeader } from "@nice-digital/nds-page-header";
+
+import { BorderlineSubstancesMenu } from "@/components/BorderlineSubstancesMenu/BorderlineSubstancesMenu";
+import { Layout } from "@/components/Layout/Layout";
+import { SectionNav } from "@/components/SectionNav/SectionNav";
+import { SEO } from "@/components/SEO/SEO";
+import { useSiteMetadata } from "@/hooks/useSiteMetadata";
+import { type SlugAndTitle } from "@/utils";
+
+import styles from "./{BnfBorderlineSubstancesTaxonomy.slug}.module.scss";
+
+export type BorderlineSubstancesSectionPageProps = {
+	data: {
+		bnfBorderlineSubstancesTaxonomy: SlugAndTitle & {
+			rootTaxonomy: {
+				childTaxonomies: {
+					title: string;
+					childTaxonomies: {
+						title: string;
+						slug: string;
+					}[];
+					slug: string;
+				}[];
+			};
+		};
+	};
+};
+
+const BorderlineSubstancesSectionPage: FC<
+	BorderlineSubstancesSectionPageProps
+> = ({
+	data: {
+		bnfBorderlineSubstancesTaxonomy: { slug, title, rootTaxonomy },
+	},
+}) => {
+	const { siteTitleShort } = useSiteMetadata();
+	const sections = rootTaxonomy.childTaxonomies;
+	const isOneLevel = sections.some(
+		(section) =>
+			section.childTaxonomies == null || section.childTaxonomies.length == 0
```

Comment:
Type def is a non-nullable list and all our gql schema are non nullable lists, so in theory you shouldn't need the null check here